pkg/core/mempool/mem_pool.go

```diff
@@ -275,6 +311,7 @@ func (mp *Pool) RemoveStale(isOK func(*transaction.Transaction) bool) {
 			}
 		} else {
 			delete(mp.verifiedMap, itm.txn.Hash())
+			mp.removeSendersFee(itm.txn)
```

Comment:
Added
```
mp.fees := make(map[util.Uint160]utilityBalanceAndFees) // it'd be nice to reuse existing map, but we can't easily clear it
```
instead of
```
newFees := make(map[util.Uint160]utilityBalanceAndFees) // it'd be nice to reuse existing map, but we can't easily clear it
```
as we should have updated `mp.fees` while calling `mp.tryAddSendersFee`